main.go

```diff
@@ -2,27 +2,22 @@ package main
 
 import (
 	"context"
-	
```

Comment:
Thanks! I'm sorry about this; I thought I'd set up my editor to always remove trailing whitespace. :flushed: